### ISSUE TITLE
importing the module makes all windows look blurry

### ISSUE THREAD

**Issue by PimWagemans09** [Akascape/CTkColorPicker#8]
ctypes.windll.shcore.SetProcessDpiAwareness(1) doesnt help.

with CTkColorPicker:
![image](https://github.com/Akascape/CTkColorPicker/assets/129096522/6a5ba558-4fea-4b9b-8018-2369ce37cd67)
without CTkColorPicker:
![image](https://github.com/Akascape/CTkColorPicker/assets/129096522/8fd83ad2-be63-4910-9679-61940eaa5a5a)
also i have set the minimum size for the window but when i import the module i cant make the window as small
as if the minimum size has increased

**Comment by PimWagemans09** [comment]
![image](https://github.com/Akascape/CTkColorPicker/assets/129096522/50b8efe7-698c-4c1f-a59a-a83f9063af02)
so i clicked "go to definition" in vs code to look inside the module and i see that ctypes.windll.shcore.SetProcessDpiAwareness(0) is run this is (i think) what causes the issue.
but why is this piece of code run?

**Comment by Akascape** [comment]
@PimWagemans09 Because of the scale purposes in display. Try to change the scale of your screen in settings and see the difference.

**Comment by PimWagemans09** [comment]
i dont see a difference except the the window is blurry in 150% scaling.
does the module work less accuratly or something?

**Comment by Akascape** [comment]
@PimWagemans09 Here is the difference you can see in my system:

<img width="600" alt="Screenshot 2023-12-11 153305" src="https://github.com/Akascape/CTkColorPicker/assets/89206401/fc1ceeba-bd29-4519-a3e3-5fca4f347d9f">

I guess I have to implement the resize feature to the color wheel.

**Comment by Akascape** [comment]
@PimWagemans09 It is fixed now, update the package. 

Actually the color wheel is a tkinter canvas widget, when used with customtkinter it creates a scaling issue. we can use that ctypes method which works for all tkinter widgets but now I found that it makes the window a little blurry. 

Luckily there are widget scalling functions available in ctk which I used to adjust the dimensions. If you have lots of other tkinter widgets, then it may become a tedious work to set the dimensions one by one.
